arcgis-ios-sdk-samples/Display information/Custom dictionary style/CustomDictionaryStyleViewController.swift

```diff
@@ -0,0 +1,71 @@
+// Copyright 2020 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import ArcGIS
+
+class CustomDictionaryStyleViewController: UIViewController {
+    /// The URL to the restaurants feature table.
+    let restaurantFeatureTableURL = URL(string: "https://services2.arcgis.com/ZQgQTuoyBrtmoGdP/arcgis/rest/services/Redlands_Restaurants/FeatureServer/0")!
+    /// The URL to the symbol style dictionary from shared resources.
+    let restaurantStyleURL = Bundle.main.url(forResource: "Restaurant", withExtension: "stylx")!
```

Comment:
Since these are each only used in one place, they would make more sense as variables at the point of use rather than properties of the view controller.